### ISSUE TITLE
Extra Block

### ISSUE THREAD

**Issue by derekc** [roelmann/moodle-theme_flexibase#95]
I think I have mentioned this before, but I cannot find a reference to it sorry.
What is the reason for this block:
![17 extra space](https://cloud.githubusercontent.com/assets/687780/18825009/ac050574-8422-11e6-937b-5a4efdb34821.png)

This is edit mode on with FB 3.2.1
-Derek

**Comment by roelmann** [comment]
That block region arose from a discussion with colleagues that they wanted someone other than site administrators to be able to add alerts.
The idea is that it is a block region, primarily intended for anyone with editing rights to add a block - usually just an html block - as an alert, and because of that function, the header of the block is deliberately removed, leaving just the block content (ie the alert).
However, there was a regression that I only found in retesting for you this morning - I've fixed it on github
https://github.com/roelmann/moodle-theme_flexibase/commit/950484a1456180d69541c3878a0a89bed6e1a9aa
and will push a bugfix version onto the plugins database (hopefully) later today.
